html/listinc.html

```diff
@@ -1,13 +1,13 @@
 {% macro editlink(link, username) -%}
     <div class="pull-right">
-      <a 
+      <a
     {% if link.edits %}
       {% set lastEditTime, lastEditName = link.lastEdit() %}
       title="edited {{ lastEditTime|time_t }} by {{ lastEditName }}"
     {% else %}
       title="edit"
     {% endif %}
-    href="/_edit_/{{ link.linkid }}{% if username %}?returnto={{ keyword }}{% endif %}"><img border="0" src="/images/edititem.gif"></a>
+    href="/_edit_/{{ link.linkid }}{% if username %}?returnto={{ keyword }}{% endif %}"><span class="glyphicon glyphicon-edit"></span></a>
```

Comment:
Another comment here fixing the missing edititem.gif, and is now using the Bootstrap edit icon.